cypress/e2e/plugincomponentdetailpanel.feature

```diff
@@ -0,0 +1,107 @@
+Feature: Test Component Detail Panel
+
+  Background:
+    Given I clear localstorage
+    And I set in localstorage field "projects" with '{"project-0001":{"id":"project-0001"}}' as "json"
+    And I visit the "/#/modelizer/project-0001/"
+    And I click on "[data-cy=\"object-details-panel-button\"]"
+
+  Scenario: Click on Object details panel button should display the panel
+    Then I expect '[data-cy="object-details-panel"]' exists
+    And I expect field '[data-cy="object-details-panel"] [data-cy="object-details-str-with-value"]' is 'TOTO'
```

Comment:
Could be better to check the static things like: name field, reset button, save button and if there is at least 1 child in attributes list.